Resources/Locale/en-US/station-events/events/zombie-outbreak.ftl

```diff
@@ -0,0 +1 @@
+station-event-zombie-outbreak-announcement = Confirmed sightings of various undead aboard the station. All personnel should arm themselves, barricade doors, and secure their location in order to prevent further infection.
```

Comment:
In SS13, Zombie virus is called Necrotic degeneration, and they are part of the diseases pool. IMO Zombie virus should be part of the disease pool with the standard "biohazard" announcement. The problem is, we got too little diseases and it might get chosen often, unless we give diseases a weight.

Reply:
A thing to note about the zombie outbreak event, it doesnt infect people like the biohazard event, but rather makes zombies outright. Besides, i feel like the thematic and gameplay systems of a disease outbreak vs zombie outbreak are different enough to warrant different events and rarities.